lua/nvim-lsp-installer/servers/taplo/init.lua

```diff
@@ -0,0 +1,15 @@
+local server = require "nvim-lsp-installer.server"
+local cargo = require "nvim-lsp-installer.installers.cargo"
+
+return function(name, root_dir)
+    return server.Server:new {
+        name = name,
+        root_dir = root_dir,
+        languages = { "toml" },
+        homepage = "https://taplo.tamasfe.dev/lsp/",
+        installer = cargo.crates { "taplo-lsp" },
+        default_options = {
+            cmd = cargo.executable(root_dir, "taplo-lsp"),
```

Comment:
![image](https://user-images.githubusercontent.com/72651387/147822199-ac659892-508a-43f3-931c-d2523ac80c77.png)

I think it should be table.

Reply:
```suggestion
            cmd = { cargo.executable(root_dir, "taplo-lsp") },
```

Reply:
```suggestion
            cmd = { cargo.executable(root_dir, "taplo-lsp"), "run" },
```

https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#taplo

Reply:
Ah thanks